torch/onnx/symbolic.py

```diff
@@ -555,6 +555,17 @@ def squeeze(g, self, dim=None):
                 dims.append(i)
     else:
         dims = [_get_const(dim, 'i', 'dim')]
+        # Handle negative dims
+        for i, dim in enumerate(dims):
+            if dim < 0:
+                if self.type().kind() == "CompleteTensorType" or self.type().kind() == "DimensionedTensorType":
+                    warnings.warn("ONNX export squeeze with negative axis " + str(dim) +
```

Comment:
Why warning? Because we suppose the rank won't change at runtime? If so, please make it clear in the warning.

Reply:
Thanks! done